ethereum_history_api/oracles/src/noir/oracles/accountOracle.int.test.ts

```diff
@@ -6,24 +6,24 @@ describe('accountOracle', () => {
   it('getAccountOracle', async () => {
     const mainnetChainIdInNoirFormat = '0x01';
     // prettier-ignore
-    const cryptoPunksAccountAddressInNoirFormat = [
-        "0xb4", "0x7e", "0x3c", "0xd8", "0x37", "0xdd", "0xf8", "0xe4", "0xc5", "0x7f",
-        "0x05", "0xd7", "0x0a", "0xb8", "0x65", "0xde", "0x6e", "0x19", "0x3b", "0xbb"
+    const vitalikAccountAddressInNoirFormat = [
```

Comment:
Good catch! Had to change it due to fact that previously crypto punks fixture wasn't containing storage proof. That proof was needed by different test case and this one was breaking. 
Discussed with @LogvinovLeon that easier would be just to use regular EOA (vitaliks address in that case) instead of smart contract address (as it's not relevant for this particular test case)